MinecraftJoypadSplitscreenMod/src/com/shiny/joypadmod/GameRenderHandler.java

```diff
@@ -239,7 +238,6 @@ private static void HandleDragAndScrolling()
 		if (VirtualMouse.isButtonDown(0) || VirtualMouse.isButtonDown(1))
 		{
 			// VirtualMouse.moveMouse(JoypadMouse.getmcX(), JoypadMouse.getmcY());
-			McGuiHelper.guiMouseDrag(JoypadMouse.getX(), JoypadMouse.getY());

```

Comment:
Thanks for submit!
We might still need this call to handle some cases.  Like when the button is down and you are moving the joystick around when scrolling through inventory sliders, other menus or some mods.